ndless/src/installer-4.5/Makefile

```diff
@@ -26,8 +26,13 @@ all: $(DISTDIR)/ndless_installer_4.5.0.tns
 %.bin: %.o
 	$(OBJCOPY) -O binary $^ $@
 
+OS_NAME := $(shell uname -s | tr A-Z a-z)
 payload: ndless_installer.bin
-	(../tools/MakeChunkDispatch/MakeChunkDispatch 0x1800e7f4 0x12480000; cat $^) | base64 -w0 > $@ || rm $@
+	ifeq ($(OS_NAME),darwin)
```

Comment:
This should be done by using feature detection instead. `base64` doesn't have/need the `-w0` option on some Linux distros either.